Common/UnitDefinitions/Molarity.json

```diff
@@ -74,7 +74,7 @@
       },
       "FromUnitToBaseFunc": "{x} / 1e-3",
       "FromBaseToUnitFunc": "{x} * 1e-3",
-      "Prefixes": [ "Pico", "Nano", "Micro", "Milli", "Centi", "Deci" ],
+      "Prefixes": [ "Femto", "Pico", "Nano", "Micro", "Milli", "Centi", "Deci" ],
```

Comment:
Hi, you need to fix the compile error by adding a test case.

```
C:\projects\unitsnet\UnitsNet.Tests\CustomCode\MolarityTests.cs(29,18): error CS0534: 'MolarityTests' does not implement inherited abstract member 'MolarityTestsBase.FemtomolesPerLiterInOneMolesPerCubicMeter.get'
```

Reply:
You can build the solution or run `dotnet build` to test that it works.
Run `build.bat` to build and run all tests from command line.

Reply:
Oh, and run `generat-code.bat`, to generate code for the new unit.

Reply:
Detailed steps here: https://github.com/angularsen/UnitsNet/wiki/Adding-a-New-Unit